Entities/AtomGroup.cpp

```diff
@@ -2307,7 +2307,7 @@ bool AtomGroup::InTerrain()
 // TODO: UNCOMMENT
     for (const Atom * aItr : m_Atoms)
     {
-        aPos = (m_pOwnerMO->GetPos() + (aItr->GetOffset().GetXFlipped(m_pOwnerMO->m_HFlipped) * m_pOwnerMO->GetRotMatrix())).GetFloored();
```

Comment:
For this and the one below, I assume something in here already floors it? I don't see anything obviously doing that, but I haven't dug into each method called in the long chain.

Reply:
It's being floored when it is used.
The vector values get converted to int when they're passed to functions anyway, so that's when the flooring happens.
Probably even better would be to use a vector of ints in the first place when we know that's how the vector will be used. And save a few operations that way.

Reply:
Okay cool, I figured one of the functions in there probably took an int, but it was late and I didn't wanna go digging. Thanks for clarifying!